examples/with-drizzle-pagesdir/src/pages/index.tsx

```diff
@@ -7,14 +7,18 @@ import { UploadButton, UploadDropzone } from "~/utils/uploadthing";
 const inter = Inter({ subsets: ["latin"] });
 
 export default function Page() {
-  const { isLoading, data } = useSWR(
+  const {
+    isLoading,
+    data: files,
+    mutate,
+  } = useSWR(
     "files",
-    () => {
-      return fetch("/api/files").then((res) => res.json()) as Promise<{
-        files: File[];
-      }>;
+    async () => {
+      const res = await fetch("/api/files");
+      const files = (await res.json()).files as File[];
+      return files;
     },
-    { refreshInterval: 300 },
+    { refreshInterval: 800 },
```

Comment:
300 seemed a bit agressive so i upped it a bit and added a manual refresh in the complete handler